install.py

```diff
@@ -208,8 +208,13 @@ def get_requirements_branch(version, inst, repository):
     """
     Convert "version" into a branch name
     """
-    if repository != 'gem/oq-engine':
+    repository_owner, repository_name = repository.split('/')
+    # in forks of oq-engine, always read requirements from master
+    if (repository_owner != 'gem' and repository_name == 'oq-engine'):
```

Comment:
The parenthesis are not needed

Reply:
Acc! I see a problem here!
I want the requirements to be read from OUR master, not from their master that could be outdated.

Reply:
According with this https://github.com/gem/oq-engine/blob/master/install.py#L393 I believe requirements are actually read from our master.